nltk/stem/arlstem.py

```diff
@@ -177,14 +171,14 @@ def pref(self, token):
             for p2 in self.pr2:
                 if(token.startswith(p2)):
                     token = token[2:]
-                    return token                        
+                    return token
         return None
-        
+
     def suff(self, token):
-        """ 
-            remove suffixes from the word's end 
         """
-        if(token.endswith('\u0643') and len(token)>3):
+            remove suffixes from the word's end.
+        """
+        if(token.endswith('\u0643') and len(token) > 3):
```

Comment:
`if` needs space after, according to PEP-8

Reply:
I didn't understand "if needs space after,"